runtime-sdk/src/context.rs

```diff
@@ -31,21 +27,126 @@ pub enum Mode {
     SimulateTx,
 }
 
+impl Mode {
+    /// Mode string representation.
+    pub fn to_str(&self) -> &'static str {
```

Comment:
So afaik to use it in the logger context as i do bellow (so that it's present in all log invocations) it needs to be [threadsafe (`'static`, `Send`, `Sync`)](https://docs.rs/slog/2.7.0/slog/struct.Logger.html#method.new). 

So I'm not sure if the display trait implementation will be useful. However I should probably move these strings into constants and use these to also implement the `Display` trait and maybe rename this method to `to_static_str` with a comment on why we need it.

Unless i missed something obvious?